ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterMapManagerTest.java

```diff
@@ -137,7 +141,7 @@ public void addNewPartition()
   }
 
   @Test
-  public void bestEffortAllocation()
+  public void bestEffortAllocationNonRackAware()
```

Comment:
I forgot to change the test name after changing the allocation method's behavior.  I'll look over these test cases and make the names more relevant.